src/components/ProgressTable/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { ProgressbarCard } from './ProgressbarCard';
+
+import { Title } from '../Title';
+
+import ProgressBarLevel1 from '@/../public/images/progressbar-bg-level-1.png';
+import ProgressBarLevel2 from '@/../public/images/progressbar-bg-level-2.png';
+import ProgressBarLevel3 from '@/../public/images/progressbar-bg-level-3.png';
+
+export const Progressbar = () => {
+  return (
+    <>
+      <Title>Progress</Title>
+      <div className="ml-9 mt-9 flex gap-x-9">
```

Comment:
add `flex-wrap` style so flex will be responsive

Reply:
It look ugly :( 
![image](https://user-images.githubusercontent.com/45608740/235977829-432332a1-637a-43cb-a061-8d43ac373c72.png)